src/notation/internal/notationselection.h

```diff
@@ -56,6 +59,7 @@ class NotationSelection : public INotationSelection
 
     IGetScore* m_getScore = nullptr;
     INotationSelectionRangePtr m_range;
+    ElementContext m_hitElementContext;
```

Comment:
I think better rename `ElementContext` to `HitElementContext`

Reply:
renamed